addon/services/log.js

```diff
@@ -0,0 +1,348 @@
+/**
```

Comment:
Actually file was just renamed, but my git client handles it as: old file - remove/new file add.